mobify/bellows
# Clicking on anchors in the header should work instead of toggling the me...

Clicking on anchors in the header should work instead of toggling the menu

Status: **Ready to merge**

Reviewers: @scalvert 
## Changes
- moved the preventDefault into the condition check so we only toggle if we match
- exit the event handler if we're clicking on or inside an anchor
## How to test-drive this PR
- checkout this branch
- `grunt test`
- `grunt serve`, ensure no linting errors, and check the examples - specifically test the default example, and bellows within bellows
- change the header DOM to this:

``` html

                            <div class="bellows__header t-sidebar__category-title-wrapper">
                                <h3 class=" t-sidebar__category-title">Dummy Header<i class="c-icon" data-icon-name="plus" aria-hidden="true"></i></h3>
                                <a href="http://google.com"><i class="c-icon t-sidebar__category-link" data-icon-name="chevron-right" aria-hidden="true"></i></a>
                            </div>
```
